### ISSUE TITLE
SMN Unlock Quest - Double Orange Light

### ISSUE THREAD

**Issue by DrfistMD** [EdenServer/community#1025]
I went to the Glacier during double ice weather and instead of receiving indigo light I received orange light a second time.  Re-zoning into areas with ice weather does not proc the indigo light cut scene and I am currently unable to complete the quest to unlock SMN.

IGN: Xerox
![unknown](https://user-images.githubusercontent.com/46073012/52917397-f0647f00-32b8-11e9-95ae-ab8a8411687a.png)

**Comment by DrfistMD** [comment]
This appears to have just been an issue with the wrong cutscene firing off.  I decided to go check the final turn in point and I was able to successfully unlock SMN.  So instead of the Carbuncle runs off and tells you to meet'em at La'Theine cutscene, I got a second Orange Light cutscene, but it did properly track my Indigo light in the background.

**Comment by Althaia** [comment]
I just finished my SMN a couple of days ago and this happened to me, too.

I can confirm it's a text-only / wrong CS issue only, because I was able to successfully complete the quest.

Additionally, this happened to me on my Water weather, not my Ice weather.

My Water weather was my second-to-last needed weather; I did Ice last (in Xarcarbard, if that's important).

Anyway, I'm not sure how the cutscenes are decided, but maybe Ice was DrfistMD's second-to-last needed weather, too? In any case, it only happened once to both of us.
